docs/tutorials/development_contributing.md

```diff
@@ -0,0 +1,372 @@
+# How to start developing with Ralph?
+
+### Prepare your environment 
+
+Welcome to our developer contribution guidelines! 
+If you're interested in contributing to our project, there are a few prerequisites to get you started. 
+Ralph development environment is containerized with Docker for consistency.
+Before diving in, ensure you have the following installed:
+
+- [Docker Engine](https://docs.docker.com/engine/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [make](https://www.gnu.org/software/make/manual/make.html)
+
+### Bootstrap Ralph for development
+
+To start playing with `ralph`, you should first `bootstrap` using:
+
+```bash
+make bootstrap
+```
+
+Once the project has been bootstrapped, you may want to edit generated `.env`
+file to set up available backend parameters that will be injected into the
+running container as environment variables to configure Ralph (see [backends documentation](../backends/index.md)):
+
+```bash
+# Elasticsearch backend
+RALPH_ES_HOSTS=http://elasticsearch:9200
+RALPH_ES_INDEX=statements
+RALPH_ES_TEST_HOSTS=http://elasticsearch:9200
+RALPH_ES_TEST_INDEX=test-index
```

Comment:
```suggestion
RALPH_BACKENDS__LRS__ES__HOSTS=http://elasticsearch:9200
RALPH_BACKENDS__LRS__ES__INDEX=statements
RALPH_BACKENDS__LRS__ES__TEST_HOSTS=http://elasticsearch:9200
RALPH_BACKENDS__LRS__ES__TEST_INDEX=test-index
```
even if it will probably change soon